app/controllers/relays_activations_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class RelaysActivationsController < ApplicationController
+
+  before_action :authenticate_user!
+  before_action :init_current_facility
+  authorize_resource :facility
+
+  def create
+    relays.each(&:activate)
```

Comment:
This is probably ok, but I know there are some facilities with lots of relays so this could be very slow (some facilities take 12-15 seconds to collect all the statuses per skylight). Would it be worth fanning out a bunch of background jobs rather than doing each one inline? Just a thought.

Reply:
One thing we’re saving on here is that we’re _not_ checking the status after doing an activate/deactivate, so I’d like to move to background jobs only if this starts to be problematic, if that’s okay!

Reply:
I checked this in the UI by signing in as ddi123@example.com, and the permissions were correct. I also changed the existing spec to [use a more restrictive Facility Director user](https://github.com/tablexi/nucore-open/pull/1848/files#diff-c24558c22a835cfca78251df8c28fcf0R7), so we’re testing the permissions as well as the behavior.